cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -48,6 +48,22 @@ Before you enable Cloud Organization SSO, notify your members about what to expe
 
 During enablement of the feature, a list of affected members is shown, and those members are also notified individually.
 
+### Ensure that at least one organization admin belongs to no other CockroachDB Cloud Organization
+
+{{site.data.alerts.callout_success}}
+Troubleshooting tip: If your migration fails with the error: `Cloud Organization SSO cannot be enabled`, confirm that at least one organization admin belongs to no other CockroachDB Cloud Organization.
+{{site.data.alerts.end}}
+
+<!--  @jason-crl I know this isn't quite right, plz help me phrase this stuff more accurately -->
+
+Under the hood, when an Organization migrates to Cloud Organization SSO from basic SSO, the identity of each member must be migrated to the new domain. If an organization members *also belongs to other CockroachDB Cloud Organizations*, their identity must be split, to create an identity on the new organization at the new SSO domain, while maintaining their identity on their other orgs at the basic SSO shared domain. This is all done automatically by the platform, and requires no action by customer organization administrators.
```

Comment:
```suggestion
Under the hood, when an Organization migrates to Cloud Organization SSO from basic SSO, the identity of each member must be migrated to the new domain. If an organization member *also belongs to other CockroachDB Cloud Organizations*, their identity must be split, to create an identity on the new organization at the new SSO domain, while maintaining their identity on their other orgs at the basic SSO shared domain. This is all done automatically by the platform, and requires no action by customer organization administrators.
```

Reply:
FWIW, I agree with @jason-crl that these are implementation details that aren't actionable by users, so they should be omitted.